deploy/crds/crd-certificates.yaml

```diff
@@ -22,28 +22,6 @@ spec:
     categories:
       - cert-manager
   scope: Namespaced
-  conversion:
-    # a Webhook strategy instruct API server to call an external webhook for any conversion between custom resources.
-    strategy: Webhook
-    # webhookClientConfig is required when strategy is `Webhook` and it configures the webhook endpoint to be called by API server.
-    webhook:
-      # We don't actually support `v1beta1` but is listed here as it is a
-      # required value for [Kubernetes v1.16](kubernetes/kubernetes#82023). The
-      # API server reads the supported versions in order, so _should always_
-      # attempt a `v1` request which is understood by the cert-manager webhook.
-      # Any `v1beta1` request will return an error and fail closed for that
-      # resource (the whole object request is rejected).
-      # When we no longer support v1.16 we can remove `v1beta1` from this list.
-      conversionReviewVersions: ["v1", "v1beta1"]
-      clientConfig:
-        # {{- if .Values.webhook.url.host }}
-        url: https://{{ .Values.webhook.url.host }}/convert
-        # {{- else }}
-        service:
-          name: '{{ template "webhook.fullname" . }}'
-          namespace: "{{ .Release.Namespace }}"
-          path: /convert
-          # {{- end }}
   versions:
     - name: v1alpha2
```

Comment:
We're removing the conversion config, but we still have the API schemas for older API versions? I'd expect that we remove the older API versions *first*, and then this configuration.

So in order (either as commits or individual PRs):

* Refactor integration tests to use testgroup
* Remove old API versions (i.e. `rm -rf ./pkg/apis/certmanager/v1{alpha,beta}*`)
* Remove this content from the CRD files

Otherwise I'm not sure how this CRD is really valid as the apiserver won't be able to convert between different versions.

Reply:
Ok. Give me a few minutes and I'll limit this PR to only updating the integration tests.

Reply:
I've updated the branch to only do the test refactoring.
I'll remove the conversion config in https://github.com/jetstack/cert-manager/pull/4635